### PR TITLE
feat: add an icon position on the left side of a basic-card

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/sub-blocks/BasicCard/BasicCard.scss [hunk 1]
```diff
@@ -4,15 +4,42 @@
 $block: '.#{$ns}basic-card';
 
 #{$block} {
+    $iconSizeL: 32px;
```
Comment (gorgeousvlad) [inline src/sub-blocks/BasicCard/BasicCard.scss:7]: why do we have L and S sizes, but don't have m?

File: src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx [hunk 2]
```diff
@@ -36,6 +37,20 @@ const WithIconTemplate: StoryFn<BasicCardProps> = (args) => (
     </div>
 );
 
+const WithIconLeftTemplate: StoryFn<BasicCardProps> = (args) => (
```
Comment (yuberdysheva) [inline src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx:40]: I think that would be nice to do one tab `with Icon` and show there default position and left position

File: src/sub-blocks/BasicCard/BasicCard.scss (hunk 1 above)
Comment (yuberdysheva) [inline src/sub-blocks/BasicCard/BasicCard.scss:7]: Why it's name is size L? 
If the size depends on position maybe we can name it something like `iconSizePositionLeft`?

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/sub-blocks/BasicCard/BasicCard.tsx [hunk 3]
```diff
@@ -5,29 +5,47 @@ import CardBase from '../../components/CardBase/CardBase';
...
 const BasicCard = (props: BasicCardProps) => {
-    const {title, text, icon, additionalInfo, links, buttons, ...cardParams} = props;
+    const {
+        title,
+        text,
+        icon,
+        additionalInfo,
+        links,
+        buttons,
+        iconPosition = IconPosition,
```
Comment (yuberdysheva) [inline src/sub-blocks/BasicCard/BasicCard.tsx:23]:
```suggestion
        iconPosition = IconPosition.Top,
```

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/sub-blocks/BasicCard/BasicCard.scss (hunk 1 above)
Comment (niktverd) [reply src/sub-blocks/BasicCard/BasicCard.scss:7]: Good point

File: src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx (hunk 2 above)
Comment (niktverd) [reply src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx:40]: Ok

File: src/sub-blocks/BasicCard/BasicCard.scss (hunk 1 above)
Comment (niktverd) [reply src/sub-blocks/BasicCard/BasicCard.scss:7]: I've changed it

File: src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx [hunk 4]
```diff
@@ -23,28 +23,14 @@ const DefaultTemplate: StoryFn<BasicCardProps> = (args) => (
     </div>
 );
 
-const WithIconTopTemplate: StoryFn<BasicCardProps> = (args) => (
+const WithIconTemplate: StoryFn<BasicCardProps> = (args) => (
```
Comment (yuberdysheva) [inline src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx]: Can we left here only two cards with specific headers?
you can see the example of borders 
https://preview.gravity-ui.com/page-constructor/481/?path=/story/components-cards-basiccard--with-border

Comment (niktverd) [reply src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx]: agree

**Review by gorgeousvlad — APPROVED** [review]

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**